cfgov/v1/models/organisms.py

```diff
@@ -137,3 +138,18 @@ class Meta:
 
     class Media:
         js = ("expandable-group.js",)
+
+
+class ItemIntroducation(blocks.StructBlock):
```

Comment:
hah i'll change that. @anselmbradford. hmm we could, can we circle back to this after the first milestone?